src/checkbox/example/index.ts

```diff
@@ -0,0 +1,62 @@
+import { WidgetBase } from '@dojo/widget-core/WidgetBase';
+import { WidgetProperties } from '@dojo/widget-core/interfaces';
+import { ProjectorMixin } from '@dojo/widget-core/mixins/Projector';
+import { StatefulMixin } from '@dojo/widget-core/mixins/Stateful';
+import { v, w } from '@dojo/widget-core/d';
+import Checkbox from '../../checkbox/Checkbox';
+
+const defaultCheckboxValues = {
+	'c1': true,
+	'c2': false,
+	'c3': false
+};
+
+const AppBase = StatefulMixin(WidgetBase);
+export class App extends AppBase<WidgetProperties> {
+	onChange(event: Event) {
+		const {
+			checkboxValues = defaultCheckboxValues
+		} = this.state;
+		const value = (<HTMLInputElement> event.target).value;
+		checkboxValues[value] = (<HTMLInputElement> event.target).checked;
+		this.setState({ checkboxValues: checkboxValues });
```

Comment:
Not super important, but you could store each checkbox value as a separate state item. That way they don't all re-render every time one changes.